src/models/github-server-app.ts

```diff
@@ -163,6 +163,53 @@ export class GitHubServerApp extends EncryptedModel {
 		});
 	}
 
+	/**
+	 * Update existing GitHubServerApp object
+	 *
+	 * @param {{
+	 * 		gitHubClientId: string,
+	 * 		uuid: string,
+	 * 		appId: number;
+	 * 		gitHubBaseUrl: string,
+	 * 		gitHubClientSecret: string,
+	 * 		webhookSecret: string,
+	 * 		privateKey: string,
+	 * 		gitHubAppName: string,
+	 * 		installationId: number
+	 * 	}} payload
+	 */
+
+	static async updateGitHubApp(payload: GitHubServerAppPayload): Promise<void> {
```

Comment:
/nit
I suggest change the name to `updateGitHubAppByUUID` so that it is explicit that `uuid` is the key to locate the record.